source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,842 @@
+      sudo ln -sf $HOME/mixer/update/www /var/www/mixer
 
-.. code-block:: bash
+#. Set up ``nginx`` configuration:
 
-   mixer versions
+   .. code-block:: bash
 
-At this point, you can continue to iterate through the workflows and make
-modifications as needed, for example:
+      sudo mkdir -p  /etc/nginx/conf.d
 
-#. Add, remove, or modify bundles.
-#. Build the chroots with:
+#. Copy the default example configuration file:
 
    .. code-block:: bash
 
-      mixer build chroots
+      sudo cp -f /usr/share/nginx/conf/nginx.conf.example /etc/nginx/nginx.conf
 
-#. Build and update with:
+#. Configure the mixer update server. Create and add the following server
+   configuration content to :file:`/etc/nginx/conf.d/mixer.conf` (sudo required):
 
-   .. code-block:: bash
+   .. code-block:: console
 
-      mixer build update
+      server {
+           server_name localhost;
+           location / {
+                     root /var/www/mixer;
+                     autoindex on;
+           }
+      }
 
-#. Optionally, you can create delta packs with:
+#. Restart the daemon, enable nginx on boot, and start the service.
 
    .. code-block:: bash
 
-      mixer build delta-packs --to <NEWVERSION> --from <PREV_VERSION>
-
-.. _mixer-format:
+      sudo systemctl daemon-reload
 
-Format version
-==============
-
-The `Format` variable set in the :file:`builder.conf` file can be more
-precisely referred to as an OS *compatibility epoch*. Versions of the OS
-within a given epoch are fully compatible and can update to any other
-version within that epoch. Across the `Format` boundary, the OS has changed
-in such a way that updating from build M in format X, to build N in format Y
-will not work. Generally, this scenario occurs when the software updater or
-software manifests change in a way that they are no longer compatible with
-the previous update scheme.
-
-Using a format increment, we insure pre- and co-requisite changes flow out
-with proper ordering. The updated client only ever updates to the latest
-release in its respective format version, unless overridden by command line
-flags. Thus, we can guarantee that all clients update to the final version
-in their given format. The given format *must* contain all the changes
-needed to understand the content built in the subsequent format. Only after
-reaching the final release in the old format can a client continue to update
-to releases in the new format.
+      sudo systemctl enable nginx
 
-When creating a custom mix, the format version should start at "1" or some
-known number such as the host system format. The format version should
-increment only when a compatibility breakage is introduced. Normal updates,
-like updating a software package for example, do not require a format
-increment.
+      sudo systemctl start nginx
 
-.. _update page: https://cdn.download.clearlinux.org/update/
+#. Verify the web server is running at \http://localhost. At this point
```

Comment:
I haven't run this update server in a while, but I want to ask if the last comment is what you intend. 
If your localhost is used as the update server, and it's restarted, shouldn't this read following?
`[...]  you should no longer see a "404 Not found" message`

Reply:
Indeed. At this point you should se the update content.